android/src/ti/map/AnnotationProxy.java

```diff
@@ -70,10 +71,12 @@
 	private static final int MSG_SET_LON = MSG_FIRST_ID + 300;
 	private static final int MSG_SET_LAT = MSG_FIRST_ID + 301;
 	private static final int MSG_SET_DRAGGABLE = MSG_FIRST_ID + 302;
-	private static final int MSG_UPDATE_INFO_WINDOW = MSG_FIRST_ID + 303;
+	private static final int MSG_SET_VISIBLE = MSG_FIRST_ID + 303;
```

Comment:
Although it shouldn't really matter in this context, I would prefer to keep `MSG_UPDATE_INFO_WINDOW` the 303 and make `MSG_SET_VISIBLE` the 304.